src/components/FoodPantryDisplay/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { FoodPantry } from 'utils/types';
+import FoodPantryContainer from 'src/components/FoodPantryContainer/index';
+import { Pagination } from '@material-ui/lab';
+
+interface Props {
+  pantries: FoodPantry[];
+}
+
+interface CardDisplayState {
+  offset: number;
+}
+
+class FoodPantryDisplay extends Component<Props, CardDisplayState> {
+  // Set default offset to 0
+  constructor(props: Props) {
+    super(props);
+    this.state = { offset: 0 };
+  }
+
+  // setState on pagination change
+  onPageChange = (event: React.ChangeEvent<unknown>, page: number) => {
```

Comment:
Add return type to remove warning
```suggestion
  onPageChange = (event: React.ChangeEvent<unknown>, page: number): void => {
```